### PR TITLE
Rework Console <-> Wallet <-> TxPool <-> Node workflow

### LINKED ISSUE #413: Rework Console <-> Wallet <-> TxPool <-> Node architeture

### PR DESCRIPTION
**Pull request by rtsisyk**
- Convert Wallet into a background service.
- Remove TxPool from Console.
- Connect Wallet <-> TxPool using unicast messages.
- Simplify TxPool logic and remove API.
- Subscribe Wallet && TxPool for epoch changes.
- Add `spay` command to CLI to send ValueShufle-enabled payments.
- Implement TxPool formation on `spay` request.

Closes #413

### CODE REVIEW

File: txpool/src/lib.rs [hunk 1]
```diff
@@ -143,11 +136,12 @@ impl TransactionPoolService {
...
                 // after timeout facilitator should broadcast message to each node.
                 let accumulator = state.take_pool();
-                let pool = PoolInfo::new(accumulator, self.pkey, &self.skey);
-                //TODO:: remove clone
-                let data = pool.clone().into_buffer()?;
-                self.network.publish(&BROADCAST_POOL_MESSAGES, data)?;
-                self.handle_receive_pool_info(pool)?;
+                let pool = PoolInfo::new(accumulator.clone(), self.pkey, &self.skey);
+                let pool = pool.clone().into_buffer()?;
+                for pkey in accumulator {
```
Comment (vldm) [inline txpool/src/lib.rs]: Why not using broadcast?

File: txpool/src/protos.rs [hunk 2]
```diff
@@ -57,15 +56,15 @@ impl ProtoConvert for PoolInfo {
...
         let pkey = secure::PublicKey::from_proto(proto.get_pkey())?;
         let sig = secure::Signature::from_proto(proto.get_sig())?;
-        let mut accumulator = Vec::new();
-        for msg in proto.get_accumulator().iter() {
-            let msg = Message::from_proto(msg)?;
-            accumulator.push(msg);
+        let mut accumulator = HashSet::new();
+        for msg in proto.get_accumulator() {
+            let pkey = secure::PublicKey::from_proto(msg)?;
+            accumulator.insert(pkey);
```
Comment (vldm) [inline txpool/src/protos.rs]: Should we return error if duplicate key was found?

File: txpool/src/lib.rs (hunk 1 above)
Comment (rtsisyk) [reply txpool/src/lib.rs]: Information about formed pool must be private.

File: txpool/src/protos.rs (hunk 2 above)
Comment (rtsisyk) [reply txpool/src/protos.rs]: Probably yes.

**Pull request merged**